setup.py

```
@@ -149,13 +149,13 @@ def run(self):
     matches = dict(VERSION_REGEX.findall(f.read()))
-    version_override = os.environ.get("SLANGPY_VERSION_OVERRIDE", "")
-    if version_override:
-        version = version_override
-    else:
-        version = "{MAJOR}.{MINOR}.{PATCH}".format(**matches)
+    version = "{MAJOR}.{MINOR}.{PATCH}".format(**matches)
+    version += os.environ.get("SLANGPY_VERSION_SUFFIX", "")
     print(f"version={version}")
 
+if BUILD_RELEASE_WHEEL and not os.environ.get("SLANGPY_VERSION_SUFFIX"):
+    raise RuntimeError("Release-wheel builds require SLANGPY_VERSION_SUFFIX")
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)